src/js/expander.js

```diff
@@ -1,12 +1,14 @@
-document.addEventListener("click", event => {
-	if (!event.target.closest(".expander button")) return false;
-	const button = event.target.closest(".expander button");
-	const expanded = button.getAttribute("aria-expanded") === "true" || false;
-	const content = button.parentNode.nextElementSibling;
-	button.setAttribute("aria-expanded", !expanded);
-	if (expanded) {
-		content.setAttribute("hidden", "true");
-	} else {
-		content.removeAttribute("hidden");
-	}
+window.addEventListener("load", () => {
+	document.querySelectorAll(".expander button").forEach(button => {
+		button.addEventListener("click", () => {
+			const expanded = button.getAttribute("aria-expanded") === "true" || false;
```

Comment:
The part of `|| false` is unnecessary. The condition in front will take care of this part.